recipes/typesystem/LICENSE.md

```diff
@@ -0,0 +1,1101 @@
+
```

Comment:
I think this license file is incorrect, right!?

Reply:
yes, totally, thank you! :)